### ISSUE TITLE
Switch off browser

### ISSUE THREAD

**Issue by itJunky** [keshavbhatt/red#10]
It's just another browser and freeze my slow system like a browser. But i want to see videos without lagged JS application in browser. How can i switch off this browser and just view videos in native interface?

**Comment by mattiasghodsian** [comment]
@itJunky check out this [project](https://github.com/TeamPiped/Piped), run it on a lightweight browser and problem solved.

**Comment by keshavbhatt** [comment]
Hi @itJunky @mattiasghodsian  This issue is fixed in latest version of Red (ver 10.0.0).

`snap install red-app` 

or update:

`snap refresh red-app`

**Comment by itJunky** [comment]
No!! Snap is stupid windows way for distributing software. I don't use it. Actually i already forget what this project is )
